test/cql-pytest/test_warnings.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+#############################################################################
+# Tests if warnings are returned in client's response
+#############################################################################
+
+from util import unique_name
+
+import pytest
+
+
+@pytest.fixture(scope="module")
+def table1(cql, test_keyspace):
+    table = test_keyspace + "." + unique_name()
+    cql.execute(f"CREATE TABLE {table} (k int primary key, t text)")
+    yield table
+    cql.execute("DROP TABLE " + table)
+
+
+@pytest.fixture(scope="module")
+def generate_batch_func(table1):
+    """generates function for batch string generation.
+    Size in kb is valid when Scylla uses default settings."""
+
+    def func(size_in_kb):
+        batch_string = "BEGIN BATCH\n"
+        statements = []
+        for idx in range(size_in_kb):
+            statements.append(f"INSERT INTO {table1} (k, t) VALUES ({idx}, '{'x'*743}')")
+        batch_string += "\n".join(statements) + "\n APPLY BATCH\n"
+        return batch_string
+    return func
+
+
+def test_warnings_are_returned_in_response(cql, table1, generate_batch_func):
```

Comment:
If I understand correctly, this test will currently fail on Scylla, so we can't merge it just yet. So either we won't merge it (maybe that's why you added it as "draft"), or we can add an "xfail" marker and merge it.

Reply:
Yes, this will currently fail on Scylla. I created this as a draft because of this and also because it's my first commit to scylla repository and I'm not sure if I'm doing it right.

Personally I prefer merging with "xfail" marker.